src/auro-flight-main.js

```diff
@@ -45,34 +51,53 @@ class AuroFlightMain extends LitElement {
     `;
   }
 
-  // function that renders the HTML and CSS into  the scope of the component
+  readTime(time) {
+    const timeDigits = time.split(' ')[0];
+    const timeFormat = time.split(' ')[1];
+
+    if (timeFormat === 'am') {
+      return `${timeDigits + ` a m`}`
+    } else {
+      return time
+    }
+  }
+
+  readStation(station) {
+    return Array.from(station).join(' ');
+  }
+
+  // Maintain content polarity between text read by screen reader and visual content.
   render() {
     return html`
-        <div class="departure">
-          <span class="departureTime">${this.departureTime}</span>
+        <div class="util_displayHiddenVisually">
```

Comment:
This hidden text needs to also mention `this.reroutedDepartureStation` and `this.reroutedArrivalStation`.

Reply:
This is a spike, not a flushed-out solution. There are a lot of missing nuances yet to be addressed.